alea/models/blueice_extended_model.py

```diff
@@ -333,6 +337,10 @@ def _build_ll_from_config(
         # Iterate through each likelihood term in the configuration
         for config in likelihood_config["likelihood_terms"]:
             blueice_config = self._process_blueice_config(config, template_folder_list)
+            for source in blueice_config["sources"]:
```

Comment:
`blueice_config["sources"]` -> `blueice_config["pdf_base_config"]["sources"]`

Reply:
@hammannr can you please explain a bit here? Thanks